framework/src/modules/chain/bft/bft.js

```diff
@@ -34,7 +34,10 @@ const extractBFTBlockHeaderFromBlock = block => ({
 	maxHeightPreviouslyForged: block.maxHeightPreviouslyForged,
 	prevotedConfirmedUptoHeight: block.prevotedConfirmedUptoHeight,
 	delegatePublicKey: block.generatorPublicKey,
-	activeSinceRound: 0, // TODO: Link the new DPOS with BFT here
+	// This parameter injected to block object to avoid big refactoring
+	// for the moment. `activeSinceRound` will be removed from the block
+	// object with https://github.com/LiskHQ/lisk-sdk/issues/4413
+	activeSinceRound: block.activeSinceRound || 0,
```

Comment:
Can we change this to activeSinceHeight? In the BFT, it should not care about the round

Reply:
Regards to this, we have decided to refactor on another issue.
@yatki Please open the issue and reference this PR